# The attitude in the aeronautical frame (right-handed, Z-down, X-front, Y-right), expressed as quaternion. Quaternion order is w, x, y, z and a zero rotation would be expressed as (1 0 0 0).
Header header
int32 ts
# Quaternion component
float32 q0
float32 q1
float32 q2
float32 q3
# Angular speed (rad/s)
float32 wx
float32 wy
float32 wz
